External/FEXCore/Source/Interface/Config/Config.json.in

```diff
@@ -38,6 +38,17 @@
           "Number of physical hardware threads to tell the process we have.",
           "0 will auto detect."
         ]
+      },
+      "CacheObjectCodeCompilation": {
+        "Type": "uint32",
+        "Default": "0",
+        "Desc": [
+          "Cache JIT object code to drive.",
+          "Allows JIT code to be shared between applications",
+          "0 is off",
```

Comment:
How about we use symbolic values there  + enum on FEXes code side? Would allow configuration to remain stable on renumbering, and also make more obvious the values.

Reply:
Switched over to textual config and enum storage.